corehq/apps/dump_reload/sql/load.py

```diff
@@ -51,7 +58,11 @@ def collect_results(dbalias_to_workerqueue) -> Tuple[list, list]:
 
         def terminate_workers(dbalias_to_workerqueue):
             for __, queue in dbalias_to_workerqueue.values():
-                queue.put(None)
+                try:
+                    queue.put(None, timeout=1)
+                except Full:
+                    # If the queue is full it's likely the worker is already terminated
```

Comment:
Is this true?  I don't have a great mental model of this, but I thought if one queue were a bit behind, the other queues would be sitting at their maxsize waiting for the slow one to process so they can accept more items.  I was imagining queues would be full most of the time.